ocs_ci/ocs/defaults.py

```diff
@@ -15,7 +15,7 @@
 # Be aware that variables defined above and below are not used anywhere in the
 # config files and their sections when we rendering config!
 
-INSTALLER_VERSION = '4.1.2'
+INSTALLER_VERSION = '4.1.4'
```

Comment:
Do we still need `INSTALLER_VERSION` and `CLIENT_VERSION` here in `defaults.py`?

Reply:
I can confirm its not required anymore, I guess its just something that escaped previous cleanups.

Reply:
Should I remove it?

Reply:
anyway, let's take care of this in future PR.
I don't want to be blocked on this

Reply:
yep makes sense, needs to be other pr with testing.